sys/compat/linuxkpi/common/include/net/mac80211.h

```diff
@@ -753,7 +753,7 @@ struct ieee80211_tx_info {
 			uint8_t				antenna;
 			uint16_t			tx_time;
 			bool				is_valid_ack_signal;
-			void				*status_driver_data[16 / sizeof(void *)];		/* XXX TODO */
+			void				*status_driver_data[2];
```

Comment:
If we wanted to upstream this to FreeBSD it might need tweaking to add padding for 32-bit, though we might also just not care about this piece of the ABI?